presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveBucketedTables.java

```diff
@@ -73,6 +90,35 @@ public void testIgnorePartitionBucketingIfNotBucketed()
                 .containsExactly(row(2));
     }
 
+    @Test(groups = {BIG_QUERY})
+    public void testIgnorePartitionBucketingIfBucketedExecutionDisabled()
```

Comment:
I assume you manually tested without disabling bucket and make sure the test failed? ;)